### ISSUE TITLE
replace "SDL2::SDL2" with "${SDL2_LIBRARIES}" in all 15 CMakeLists.txt files

### ISSUE THREAD

**Issue by beaumanvienna** [PacktPublishing/Mastering-Graphics-Programming-with-Vulkan#48]

**Comment by theWatchmen** [comment]
I think this issue might come down to the CMake and/or SDL version installed on your system. Which versions and OS are you using?

**Comment by bkolligs** [comment]
I had this problem on Ubuntu 20.04 but it doesn’t seem to be an issue on Ubuntu 22.04. However I believe @beaumanvienna uses Zorin which is based off of Ubuntu.

**Comment by beaumanvienna** [comment]
yep, I use Zorin, which is based on Ubuntu 20.04, cmake version 3.16.3

If you have a newer cmake version, can you try what I suggested?

**Comment by theWatchmen** [comment]
I haven't forgotten about this, I simply haven't had a chance to test it yet :) I should be able to do so in the next few days.
